src/hobbygroups/components/HobbyCard/Title.tsx

```diff
@@ -0,0 +1,5 @@
+import React, { FC } from 'react';
+
+import style from './card.less';
+
+export const Title: FC = ({ children }) => <h2 className={style.title}>{children}</h2>;
```

Comment:
Again, does this need it's own file? Especially when you have the same `less`-document

Reply:
answered in another comment